modules/collection/programs/hyprpanel.nix

```
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, rumLib, ... }: let
+  inherit (lib.modules) mkIf;
+  inherit (lib.options) mkOption mkEnableOption mkPackageOption;
+  inherit (lib.generators) toJSON;
+  inherit (lib.types) nullOr either str path attrs;
+
+  cfg = config.rum.programs.hyprpanel;
+in {
+  options.rum.programs.hyprpanel = {
+    enable = mkEnableOption "HyprPanel";
+
+    package = mkPackageOption pkgs "hyprpanel" {};
+
+    settings = mkOption {
+      type = attrs;
+      default = {};
+      example = {
+        bar = {
+          autoHide = "never";
+          location = "top";
+        };
+        theme = {
+          name = "catppuccin-mocha";
+          font = "JetBrains Mono Nerd Font";
+        };
+      };
+      description = ''
+        Configuration written to {file}`$HOME/.config/hyprpanel/config.json`.
+        Refer to [HyprPanel documentation](https://hyprpanel.com/configuration).
```

This link 404s, I think you meant to use: https://hyprpanel.com/configuration/settings.html
```suggestion
        Please reference [hyprpanel's documentation]

        [hyprpanel's documentation]: https://hyprpanel.com/configuration/settings.html
```

---

Can you please revisit this and your other comment to check wether they still apply?